doc/developer-guide/internal-libraries/buffer-writer.en.rst

```diff
@@ -353,7 +364,7 @@ arguments in generating out in the buffer. The basic format is divided in to thr
       min: integer
       precision: integer
       max: integer
-      type: "x" | "o" | "b"
+      type: "g" | "x" | "X" | "d" | "o" | "b" | "B" | "p" | "P"
 
```

Comment:
Should min / precision / max be described as positive integers?  Does 'fill' default to the blank char?

Reply:
Non-negative integers. Yes, the fill char defaults to space. I thought that was mentioned.